src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
@janvorli Both ARM and x86 defines ``RUNTIME_FUNCTION__SetUnwindInfoAddress`` in the same way::
```
#define RUNTIME_FUNCTION__SetUnwindInfoAddress(prf,address) do { (prf)->UnwindData = (address); } while (0)
```
Is there any contract between unwinder and runtime? I couldn't understand why ``RtlpGetFunctionEndAddress`` for ARM works...